SPECS/glibc/CVE-2025-0395.patch

```diff
@@ -0,0 +1,61 @@
+From 7bea1ae667498e404999793a27b3c990cc3e1926 Mon Sep 17 00:00:00 2001
+From: Kanishk Bansal <kanbansal@microsoft.com>
+Date: Thu, 22 May 2025 17:24:58 +0000
+Subject: [PATCH] CVE-2025-0395
+
+Upstream Patch Reference : https://sourceware.org/git/gitweb.cgi?p=glibc.git;h=8b5d4be762419c4f6176261c6fea40ac559b88dc
```

Comment:
Can we also include the tests for this CVE you've added for 3.0: https://github.com/bminor/glibc/commit/f984e2d7e8299726891a1a497a3c36cd5542a0bf?